src/components/ChatLog.jsx

```diff
@@ -0,0 +1,18 @@
+import * as PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry.jsx';
+import {useState} from 'react';
+
+const ChatLog = ({entries}) => {
+  const [numOfLikes, setLikes] = useState(entries?.filter(i => i.liked === true)?.length);
+  return <div>
+    <div>
+      {`${numOfLikes} ❤️s`}
+    </div>
+    {
+      entries?.map(i => <ChatEntry key={i.id} body={i.body} sender={i.sender} timeStamp={i.timeStamp} liked={i.liked} numOfLikes={numOfLikes} setLikes={setLikes}
```

Comment:
This line is challenging to read and violates the JS styleguide that suggests each line is no more than 120 characters long.

Using new lines for each prop being passed would help with readability.

Reply:
Prefer a more descriptive name than `i`. Maybe `entry`?